src/components/DayButton.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const DayButton = ({ text, active, onPress, key, id }) => {
```

Comment:
Remove `key` and `id` from here, cause `key` in prop causes warning in console, and we want console to be clean. Also `id` as a prop doesn't do anything, we just need it in parent component to check which day is active.